initrd/bin/oem-factory-reset

```diff
@@ -354,6 +378,22 @@ report_integrity_measurements()
     fi
 }
 
+usb_security_token_capabilities_check()
+{
+    TRACE "Under /bin/oem-factory-reset:usb_security_token_capabilities_check"
+
+    enable_usb
+    # ... first set board config preference
+    if [ -n "$CONFIG_GPG_ALGO" ]; then
+      GPG_ALGO=$CONFIG_GPG_ALGO
+			DEBUG "Setting \$GPG_ALGO to (board-)configured: $CONFIG_GPG_ALGO"
+    fi
+    # ... overwrite with usb-token capability
+    if lsusb | grep -q "20a0:42b2"; then
+      GPG_ALGO="p256"
+      DEBUG "Nitrokey 3 detected: Setting \$GPG_ALGO to: $GPG_ALGO"
```

Comment:
This I think should be a standard echo: everyone should see the override happen.

@JonathonHall-Purism ?

Here again no need to escape $GPG_ALGO, but you might want this to be useful and not show p256 to : p256 but say that GPG algo used will be p256 so

echo "Nitrokey 3 detected: Setting GPG algo to $GPG_ALGO"

or switch the echo and setting variable so that you can

echo "Nitrokey 3 detected: switching GPG algo from: $"

Reply:
keeping DEBUG, removing '$'